text/src/autogluon/text/text_prediction/predictor/predictor.py

```diff
@@ -292,60 +318,87 @@ def fit(self,
+                                                    f' but there is ' \
+                                                    f'no such column in the DataFrame.'
+            model_hparams = hyperparameters['models']['MultimodalTextModel']
+            if plot_results is None:
+                plot_results = in_ipynb()
             self._model.train(train_data=train_data,
                               tuning_data=tuning_data,
                               num_cpus=num_cpus,
                               num_gpus=num_gpus,
                               search_space=model_hparams['search_space'],
                               tune_kwargs=hyperparameters['tune_kwargs'],
                               time_limit=time_limit,
+                              continue_training=True,
                               seed=seed,
                               plot_results=plot_results,
                               verbosity=verbosity)
         else:
-            raise NotImplementedError("Currently, we only support using "
-                                      "the autogluon-contrib-nlp and MXNet "
-                                      "as the backend of AutoGluon-Text. In the future, "
-                                      "we will support other models.")
+            column_types, problem_type = infer_column_problem_types(train_data, tuning_data,
+                                                                    label_columns=label_columns,
+                                                                    problem_type=self._problem_type,
+                                                                    provided_column_types=column_types)
+            self._eval_metric, log_metrics = infer_eval_log_metrics(problem_type=problem_type,
+                                                                    eval_metric=self._eval_metric)
+            has_text_column = False
+            for k, v in column_types.items():
+                if v == _C.TEXT:
+                    has_text_column = True
+                    break
+            if not has_text_column:
+                raise AssertionError('No Text Column is found! This is currently not supported by '
+                                     'the TextPredictor. You may try to use '
+                                     'autogluon.tabular.TabularPredictor.\n'
+                                     'The inferred column properties of the training data is {}'
+                                     .format(column_types))
+            logger.info('Problem Type="{}"'.format(problem_type))
+            logger.info(printable_column_type_string(column_types))
+            self._problem_type = problem_type
+            if 'models' not in hyperparameters or 'MultimodalTextModel' not in hyperparameters['models']:
+                raise ValueError('The current TextPredictor only supports "MultimodalTextModel" '
```

Comment:
Do we need to clarify somewhere that MultimodalTextModel should be used even for data with only text? Otherwise users may find this confusing